src/endpoint/iam/ops/iam_get_user.js

```diff
@@ -0,0 +1,47 @@
+/* Copyright (C) 2024 NooBaa */
+'use strict';
+
+const dbg = require('../../../util/debug_module')(__filename);
+const iam_utils = require('../iam_utils');
+const { FORM_URL_ENCODED } = require('../../../util/http_utils');
+
+/**
+ * https://docs.aws.amazon.com/IAM/latest/APIReference/API_GetUser.html
+ */
+async function get_user(req, res) {
+
+    const params = {
+        username: req.body.user_name,
+    };
+    dbg.log1('IAM GET USER', params);
+    const reply = await req.account_sdk.get_user(params);
+    dbg.log2('get_user reply', reply);
+
+    return {
+        GetUserResponse: {
+            GetUserResult: {
+                User: {
+                    UserId: reply.user_id,
+                    Path: reply.path ?? iam_utils.AWS_EMPTY_PATH,
```

Comment:
are you sure that `??` should be used here and not `||` ?
in other words - if `reply.path === ''` should we return `''` or `AWS_EMPTY_PATH`?

Reply:
I will change the name of `AWS_EMPTY_PATH` to `AWS_DEFAULT_PATH` so it will be clearer.
The suggested path `''` is illegal, the path should either start with `/` or start and end with `/` (see [here](https://docs.aws.amazon.com/IAM/latest/APIReference/API_CreateUser.html)):

> a string of characters consisting of either a forward slash (/) by itself or a string that must begin and end with forward slashes

Reply:
so should you also use || instead of ?? so that it will fallback also on empty string?

Reply:
Only because I didn't add the user validation in this PR... I will change it from `??` to `||`.